src/classes/BDI_DataImportFLSService.cls

```diff
@@ -155,6 +156,13 @@ public class BDI_DataImportFLSService {
         }
     }
 
+    @TestVisible
+    private Boolean isValidateUpdateException(SObjectType objectType, SObjectField field){
+            return (!(objectType == Campaign.SObjectType
```

Comment:
Should this be the CampaignMember SObjectType?

Reply:
Yes it should be CampaignMember